api/Window.json

```diff
@@ -5429,6 +5429,54 @@
               "deprecated": true
             }
           }
+        },
+        "popup_feature": {
```

Comment:
The situation here is a bit complex because the other features here aren't structured very conventionally. Sorry about that.

The way forward here is to rename this feature as `features_parameter_popup` and a description like `<code>features</code> parameter accepts <code>"popup"</code> value`(following [the convention documented in our data guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#parameters-and-parameter-object-features)). You'll probably have to re-sort the file here, too.

(The MDN page wrongly identifies the parameter as `windowFeatures`, but let's overlook that for now.)